static/src/javascripts/projects/commercial/modules/dfp/render-advert.js

```diff
@@ -193,10 +193,18 @@ export const renderAdvert = (
             const callSizeCallback = () => {
                 if (advert.size) {
                     let size = advert.size.toString();
+
                     if (size === '0,0') {
                         size = 'fluid';
                     }
 
+                    /**
+                     * we reset hasPrebidSize to the default
+                     * value of false for subsequent ad refreshes
+                     * as they may not be prebid ads.
+                     * */
+                    advert.hasPrebidSize = false;
```

Comment:
Here as well, a test might be a great help. Great comment, but it's best to have comments be represented in tests so the computer can catch them.